src/Orleans.Serialization/Buffers/Adaptors/PoolingStreamBufferWriter.cs

```diff
@@ -89,7 +89,7 @@ private void Resize(int sizeHint)
         }
 
         /// <inheritdoc />
-        public void Dispose()
+        public readonly void Dispose()
```

Comment:
This is misleading and should be reverted - the struct is actually unusable after `Dispose` is called.